Changes

```diff
@@ -137,7 +137,8 @@ Working version
 - #11446: document switch compilation (lambda/switch.ml)
   (Gabriel Scherer, review by Luc Maranget and Vincent Laviron)
 
-- #11601, #11612: Clean up some global state handling in emitcode, spill.
+- #11601, #11612, #11628: Clean up some global state handling
+  in emitcode, bytepackager, spill.
   (Hugo Heuzard, Stefan Muenzel review by Vincent Laviron and Gabriel Scherer)
```

Comment:
@hhugo could you add "Nathanaëlle Courant" to the reviewer list?

Reply:
done